vignettes/web_only/repFilter_v3.Rmd

```diff
@@ -91,82 +91,74 @@ Use  `include` and `exclude` options to select a subset of samples or repertoire
 
 Let's filter your data by metadata `immdata` datasets when *Status* is *C* and look at new meta data:
 ```{r example-1.1}
-# method = "by.meta", query = list(Status = include("C"))
-immdata$meta %>% filter(Status == 'C')
+repFilter(immdata, .method = "by.meta", .query = list(Status = include("C")))$meta
 ```
-This way we have filtered both metadata and data:
+
+Note that we filter both metadata and data:
 
 ```{r example-1.1.1}
 # filtered data
-names(immdata$data)
+names(repFilter(immdata, .method = "by.meta", .query = list(Status = include("C")))$data)
 ```
 
 Filter out samples where *Lane* is not A:
 
 ```{r example-1.2}
-# method = "by.meta", query = list(Lane = exclude("A"))
-immdata$meta %>% filter(Status != 'C')
-names(immdata$data)
+repFilter(immdata, .method = "by.meta", .query = list(Lane = exclude("A")))$meta
+names(repFilter(immdata, .method = "by.meta", .query = list(Lane = exclude("A")))$data)
 ```
 
 Filter for samples where *Lane* is *B* or *C*:
 
 ```{r example-1.3}
-# method = "by.meta", query = list(Lane = include("B", "C"))
-immdata$meta %>% filter(Lane == 'C' | Lane == 'B')
-names(immdata$data)
+repFilter(immdata, .method = "by.meta", .query = list(Lane = include("B", "C")))$meta
+names(repFilter(immdata, .method = "by.meta", .query = list(Lane = include("B", "C")))$data)
 ```
 
-Filter out samples where  *Lane* is not *A* and not *C*
+Filter out samples where  *Lane* is not *A* and not *C*:
 
 ```{r example-1.4}
-# method = "by.meta", query = list(Lane = exclude("B", "C"))
-immdata$meta %>% filter(Lane != 'C' & Lane != 'A')
-names(immdata$data)
+repFilter(immdata, .method = "by.meta", .query = list(Lane = exclude("A", "C")))$meta
+names(repFilter(immdata, .method = "by.meta", .query = list(Lane = exclude("A", "C")))$data)
 ```
 
 #### Example 2
-Use  `interval`, `lessthan`(from minus infinity to your value) or `morethan`(from your value to plus infinity) to define interval values in the filter statement. These options accept float.
+Use  `interval`, `lessthan`(from minus infinity to your value) or `morethan`(from your value to plus infinity) to define interval values in the filter statement. These options accept float. `Lessthan` and `morethan` do not include borders and `interval` includes the left border and excludes the right border.
```

Comment:
```suggestion
Use  `interval`, `lessthan`(from minus infinity to your value) or `morethan`(from your value to plus infinity) to define interval values in the filter statement. These options accept float. `lessthan` and `morethan` do not include borders and `interval` includes the left border and excludes the right border.
```

Reply:
Don't use capital letters for text in backticks, because backticks mean fragments of code